gluten-ut/common/src/test/scala/org/apache/spark/sql/GlutenSQLTestsBaseTrait.scala

```diff
@@ -30,6 +31,10 @@ import org.scalatest.Tag
 /** Basic trait for Gluten SQL test cases. */
 trait GlutenSQLTestsBaseTrait extends SharedSparkSession with GlutenTestsBaseTrait {
 
+  protected def testWithGlutenPrefix(testName: String, testTag: Tag*)(testFun: => Any)(implicit
```

Comment:
Is `testWithGlutenPrefix` a little bit tedious? Probably we can use `glutenTest` or something?

Reply:
I felt it may be better to name the method starting with "test". If we have some other better name, we can fix it in the future.

Reply:
Or probably `testGluten` ? It's just nit so I'll be fine to all names anyway. 

Actually naming with term `prefix` exposes the internal rationale of this API to user. One may not know we are distinguish the tests with name prefix after the PR gets merged. That's a reason why I was thinking of a name changing.

Reply:
+ 1 for testGluten, if no other comments, i will use this to replace `testWithGlutenPrefix`.

Reply:
@zhztheplayer, @zwangsheng, ok to me! Thanks!